examples/riot/coap_federated/receiver/Makefile

```diff
@@ -0,0 +1,43 @@
+# name of your application
+APPLICATION = lf-coap-federated-receiver
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../../../../../RIOT
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# Comment this out to disable code in RIOT that does safety checking
+# which is not needed in a production environment but helps in the
+# development process:
+DEVELHELP ?= 1
+
+# Change this to 0 show compiler invocation lines by default:
+QUIET ?= 1
+
+# Enable reactor-uc features
+# CFLAGS += -DNETWORK_CHANNEL_TCP_POSIX
+CFLAGS += -DNETWORK_CHANNEL_COAP_RIOT
+REACTION_QUEUE_SIZE = 32
+EVENT_QUEUE_SIZE = 32
+
+CFLAGS += -DTHREAD_STACKSIZE_DEFAULT=10000
+CFLAGS += -DTHREAD_STACKSIZE_MAIN=10000
+CFLAGS += -DISR_STACKSIZE=10000
+
+# Configure CoAP retransmission timeout
+CFLAGS += -DCONFIG_GCOAP_NO_RETRANS_BACKOFF=1
+CFLAGS += -DCONFIG_COAP_ACK_TIMEOUT_MS=400
+CFLAGS += -DCONFIG_COAP_MAX_RETRANSMIT=4
+
+# Set the remote sender address
+ifndef REMOTE_ADDRESS
+ifndef ONLY_GET_IP
+  $(error REMOTE_ADDRESS is not defined. Please define it!)
+else
```

Comment:
Is this correct Make?

Reply:
I have rewritten this. I think it is allowed, but I made it unnecessarily complicated.